be/CMakeLists.txt

```diff
@@ -193,6 +195,11 @@ set_target_properties(curl PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/li
 add_library(lz4 STATIC IMPORTED)
 set_target_properties(lz4 PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/liblz4.a)
 
+if (${WITH_LZO} STREQUAL "ON")
```

Comment:
It will be better to use "$ENV{WITH_LZO}" 
Then there is no need to change build.sh and run-be-ut.sh